content/quiz_events.md

```diff
@@ -4,21 +4,134 @@
+    - This means the protest must apply to the question and ruling that just happened.
+- The coach may only confer with their assistant coach and substitute quizzer, before protesting.
+- The quiz officials and coaches will leave the room and attempt to come to an agreement.  If no agreement is reached within 10 minutes, the meet director will arbitrate and determine the just solution.
+- If a protest is launched after question number 20 is completed it must be done immediately after the ruling is announced.
+
+Broader, logistical issues do not require a Protest. Examples include:
+- If a quizzer or coach feels there is an equipment issue.
+- If the wrong participants are in the quiz.
+- If the quiz is at a point where Question Type Minimums or Maximums cannot be met.
+
+In those broader logitical situations, a quizzer or coach should bring the issue to the attention of the quiz officials, and attempts should be made to test, verify/reject, and rectify the situation.
 
 ## Fouls
+- Fouls may be called by any quiz official:  the quizmaster, answer judge, or scorekeeper.
+- A foul must be called before the quizmaster calls for the next question.
+- If the officials fail to call a foul, a coach or team captain may call it to the attention of the quizmaster.  If in the quizmaster's (and/or the judge's) opinion a foul might have been committed, the quizmaster will give due warning to the violating team.  A foul may not be charged, and points deducted when the quizmaster receives such help from a coach or captain.
+- The quizzer upon whom a foul is called becomes ineligible to answer that numbered question, including the A and B questions.
+- Three fouls by a quizzer eliminate that quizzer from the quiz.  The quizzer must leave the platform immediately.  His/her seat will continue to be ineligible to jump for the entire numbered question including A and B.
+
+The following are fouls:
+- Team members (quizzers, coaches) communicating with each other, other teams, or with the audience between the time the question type is introduced, and points are awarded. 
+     - Communication can include non-verbal communication.
+- The use of the hands to assist in jumping.  Hands may not touch any part of the chair or the floor from the time "Question" is called until the quizmaster recognizes the first one up.
+- When electronic equipment is used, if any light goes on after the quizmaster calls "Question" and before they begin to discernibly read the question.
+    - Discernibly includes the quizmaster's mouth forming a word shape.
+- During a Reference question or Quote question, when electronic equipment is used, if any light goes on after the quizmaster calls “Question” and before they begin to discernibly read the chapter number.
+- Starting to answer before being recognized by the Quizmaster.
+- Having more than the designated team members, one coach, and an assistant coach on the quiz platform during a time-out.
+- Challenging after conferring with anyone
```

Comment:
Should a line like this exist both under Fouls and under Challenges?

Reply:
I think it's fine in this case, because we want to have a comprehensive list of foul-able behavior, while also indicating under the Challenges section a full account of what happens in this scenario.

Reply:
I agree with Swingle.